package.json

```
@@ -40,10 +40,10 @@
   "scripts": {
     "test": "npm run lint && npm run test-unit",
     "lint": "standard",
-    "test-unit": "node_modules/.bin/mocha test/common/unit.js",
-    "test-tedious": "node_modules/.bin/mocha test/tedious",
-    "test-msnodesqlv8": "node_modules/.bin/mocha test/msnodesqlv8",
-    "test-cli": "node_modules/.bin/mocha test/common/cli.js"
+    "test-unit": "node_modules/.bin/mocha -t 15000 test/common/unit.js",
```

If these are unit tests then maybe we should stick to 5:
```suggestion
    "test-unit": "node_modules/.bin/mocha -t 5000 test/common/unit.js",
```